OOP-Assignment1/1. Calculator/Calculator.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * 
+ * @author Preeti Kumawat
+ * Date: 20-07-2017
+ * Class Name: Calculator
+ *
+ */
+public class Calculator {
+	private static double operand1;
```

Comment:
Static variable not required to maintain the state for button class. There should be a calculator instance in button class instead of this.